docs/auth.md

```diff
@@ -0,0 +1,191 @@
+# Serving Certificates, Authentication, and Authorization
+
+This document outlines how the service catalog API server handles
+authentication and authorization, and the steps needed to set this up in
+your cluster.
+
+Additionally, it provides instructions for generating the serving
+certificates used to securely serve the service catalog API.
+
+## Quickstart
+
+1. Ensure that you have a ConfigMap in the `kube-system` namespace named
```

Comment:
Maybe add the kubectl cmd? 
`kubectl get configmap --namespace kube-system extension-apiserver-authentication -o yaml` ?